ovh/resource_iam_resource_group.go

```diff
@@ -0,0 +1,152 @@
+package ovh
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceIamResourceGroup() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"resources": {
+				Type:     schema.TypeSet,
+				Required: true,
```

Comment:
```suggestion
				Optional: true,
```

Reply:
I was unsure about this one, since there is not really a meaning to an empty resource group, but since the API say it's OK I changed it.